### ISSUE TITLE
Tray icon unresponsive

### ISSUE THREAD

**Issue by StupidDuck** [ipfs-shipyard/ipfs-desktop#1522]
- OS: Ubuntu 20.04, fresh install, updated
 - Version of IPFS Desktop 0.11.4 (snap)

**Describe the bug**
The icon tray does not respond to click event

**Expected behavior**
Context menu should appear

**Additional context**
Tested on a newly installed ubuntu 20.04, all updated, Gnome 3.36.2

**Comment by jayschwa** [comment]
Possible upstream issue: https://github.com/electron/electron/issues/21576

**Comment by jayschwa** [comment]
A fix has allegedly been merged into Electron (and backported to older release branches). https://github.com/electron/electron/pull/23674

**Comment by jayschwa** [comment]
I experience this issue only when using the snap package. The tray icon menu works when using the Debian package.

**Comment by Azulan** [comment]
I also have the same issue using the snap package on Lubuntu 20.04.

**Comment by jessicaschilling** [comment]
@lidel and/or @rafaelramalho19 - FYI, thanks!

**Comment by SvenMeyer** [comment]
**System** : Manjaro / Arch Linux + cinnamon
**Version** : ipfs-desktop 0.11.3-1 (from AUR)

During the last weeks I had strange router chrashes and freeze-ups, spent hours with ISP support, but only after I installed "Net Activity Viewer", I realized that there were still 3 IPFS destops, ~30 ipfs processes running, and 300 open network connections made by IPFS (so it looks like the cheap ISP router could not handle the amount of netcwork connections - of which I was not aware all the time.)
I haven't seen a tray icon for a week, and if I re-started ipfs-desktop, I just got another process running with no way to close all ipfs processes and network connections ...

Looks like the whole "tray icon issue" is a mess and maintainer of the Arch Linux AUR package already submitted a pull request.
https://github.com/develar/app-builder/pull/29

**Comment by lidel** [comment]
Correct, Electron team broke tray support by merging Chromium changes, then backed out of them,  in the meantime some distros and desktop environments introduced workaround and patches, some did not, tray icon UX and DX in Linux ecosystem is beyond bad right now.

We are tracking this already in https://github.com/ipfs-shipyard/ipfs-desktop/issues/1153#issuecomment-634293717 


---- 

However,  I believe the current issue is about separate problem:

-  "starting ipfs-desktop more than once (via icon or cli) spawns more than one daemon"

I believe Desktop should detect its already running, and show WebUI instead of spawning new instance. I vaguely remember us fixing that in the past, but perhaps the fix was not generic enough?

**Comment by SvenMeyer** [comment]
https://github.com/ipfs-shipyard/ipfs-desktop/issues/1611

**Comment by lidel** [comment]
This should be fixed in [v0.13.2](https://github.com/ipfs-shipyard/ipfs-desktop/releases/tag/v0.13.2).
If there are any new issues with tray, please open a new issue.
